applications/display-server/guithread.cpp

```
@@ -12,46 +12,39 @@
+#include <setjmp.h>
+#include <sys/mman.h>
 
 void GUIThread::run(){
     O_DEBUG("Thread started");
     clearFrameBuffer();
     QTimer::singleShot(0, this, [this]{
         Q_ASSERT(QThread::currentThread() == (QThread*)this);
-        m_repaintMutex.lock();
-        forever{
+        QMutexLocker locker(&m_repaintMutex);
+        Q_UNUSED(locker);
```

Cppcheck (2.10-2)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_CLASSINFO is a macro then please configure it.

(unknownMacro)